fontforge/lookups.c

```diff
@@ -5041,6 +5041,9 @@ return( xasprintf( _("%s is not a class name for the forward classes." ), parsed
 	}
       break;
       default:
+        for ( i=0; i<cnt; ++i )
+	    free( parsed[i].entity );
```

Comment:
One would have to read the algorithm above very closely to know whether an `entity` could wind up NON-NULL at this point, so freeing them here makes sense, given that whenever one is "used" at the end the pointer is set to NULL. 

Similarly, one would have to read the algorithm above very closely to know whether a `replacements` could wind up non-NULL at this point, so freeing them makes just as much sense (to me), also given that whenever one is "used" the pointer is set to NULL. 

So either you've got a better understanding of the algorithm (which is quite possible) or you could also free `replacements` here).

Reply:
Thank you for your feedback, I reverted this part.

Reply:
My comment was probably too rhetorical. It should be fine to just change it to:
```
for ( i=0; i<cnt; i++ ) {
    free( parsed[i].entity );
    free( parsed[i].replacements );
}
```
I was just nattering on about why you might see one of those lines without the other. But even if there are no `replacements` to free because of how the loop above works, this will be fine -- they'll all be NULL).

Reply:
No pb, as I said I'm just trying to fix low hanging fruits, there are already some Coverity reports about Resource leak I put aside because it needs a real understanding of the code.